classifier.go

```diff
@@ -0,0 +1,96 @@
+package slinguist
+
+import (
+	"math"
+
+	"gopkg.in/src-d/simple-linguist.v1/internal/tokenizer"
+)
+
+// GetLanguageByClassifier takes in a content and a list of candidates, and apply the classifier's Classify method to
+// get the most probably language. If classifier is null then DefaultClassfier will be used.
+func GetLanguageByClassifier(content []byte, candidates []string, classifier Classifier) string {
+	if len(content) == 0 {
+		return OtherLanguage
+	}
+
+	if classifier == nil {
+		classifier = DefaultClassifier
+	}
+
+	languages := make([]string, 0, len(candidates))
+	for _, candidate := range candidates {
+		lang := GetLanguageByAlias(candidate)
```

Comment:
Maybe a better interface for `string GetLanguageByAlias(string)` could be `string,bool GetLanguageByAlias(string)`. Doing this, you don't need to return and check by "OtherLanguage".